chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -370,6 +370,25 @@ section-related =
 section-attachment-info =
     .label = { pane-attachment-info }
 
+section-button-remove =
+    .tooltiptext = Remove
+section-button-add =
+    .tooltiptext = Add new
+section-button-expand =
+    .tooltiptext = Expand
+section-button-collapse =
+    .tooltiptext = Collapse
```

Comment:
Remove, Expand, and Collapse can be `general-` strings at the top of the file and then pulled in as variables.

Reply:
For sure - I moved them to `general-*` variables at the top of .ftl file